--- korganizer-17.11.90/src/akonadicollectionview.cpp.omv~	2017-12-03 18:30:59.974671113 +0100
+++ korganizer-17.11.90/src/akonadicollectionview.cpp	2017-12-03 18:31:18.387713473 +0100
@@ -577,7 +577,7 @@ AkonadiCollectionView::AkonadiCollection
 #else
     QSortFilterProxyModel *filterTreeViewModel = new QSortFilterProxyModel(this);
     filterTreeViewModel->setFilterCaseSensitivity(Qt::CaseInsensitive);
-    filterTreeViewModel->setRecursiveFiltering(true);
+    filterTreeViewModel->setRecursiveFilteringEnabled(true);
     filterTreeViewModel->setDynamicSortFilter(true);
     filterTreeViewModel->setSourceModel(searchProxy);
 //   filterTreeViewModel->setObjectName( "Recursive filtering, for the search bar" );
