diff -up korganizer-20.03.90/src/dialog/noteeditdialog.cpp.omv~ korganizer-20.03.90/src/dialog/noteeditdialog.cpp
--- korganizer-20.03.90/src/dialog/noteeditdialog.cpp.omv~	2020-04-13 01:47:28.808723429 +0200
+++ korganizer-20.03.90/src/dialog/noteeditdialog.cpp	2020-04-13 01:47:38.720834214 +0200
@@ -87,17 +87,10 @@ NoteEditDialog::NoteEditDialog(QWidget *
                                                       "Calendar where the new note will be stored."));
 #endif
     mCollectionCombobox->setToolTip(i18n("Calendar where the new note will be stored."));
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mCollectionCombobox,
             QOverload<int>::of(
                 &Akonadi::CollectionComboBox::currentIndexChanged), this,
             &NoteEditDialog::slotCollectionChanged);
-#else
-    connect(mCollectionCombobox,
-            QOverload<int, const QString &>::of(
-                &Akonadi::CollectionComboBox::currentIndexChanged), this,
-            &NoteEditDialog::slotCollectionChanged);
-#endif
     connect(mCollectionCombobox, QOverload<int>::of(
                 &Akonadi::CollectionComboBox::activated), this,
             &NoteEditDialog::slotCollectionChanged);
